wiki/Beatmapping/Beatmap_submission/de.md

```diff
@@ -0,0 +1,27 @@
+# Einreichen
+
+[Beatmaps](/wiki/Beatmap) können auf die osu!-Webseite über den [Editor im Spiel](/wiki/Client/Beatmap_editor) eingereicht werden. Durch das Einreichen von Beatmaps können andere Nutzer ein Augenmerk auf diese werfen und die Beatmaps möglicherweise in die Kategorien [Ranked](/wiki/Beatmap/Category#ranked) oder [Loved](/wiki/Beatmap/Category#loved) eintreten.
+
+![](img/bss_warning-DE.png "Bildschirm des Beatmap-Einreichungssystems")
+
+Beim Anwählen von `Beatmap hochladen ...` aus dem Dropdown `Datei` im Editor (Kürzel: `Strg` + `Shift` + `U`) öffnet sich das Fenster für das **Beatmap-Einreichungssystem** (auch **Beatmap Submission System** oder ***BSS*** genannt). Hier werden zu Anfang Informationsquellen aufgelistet, um Nutzern zu helfen, Fragen im Bezug auf Mapping zu stellen, Feedback für ihre Beatmaps zu finden und dass ihre Beatmaps für das Ranking geeignet sind. Solltest du Probleme mit dem BSS haben, dann siehe den Ratgeber über [Probleme mit dem BSS](/wiki/Guides/BSS_Issues).
+
+Wenn die Beatmap, die ein Nutzer hochlädt, nicht bereits auf der osu!-Webseite ist, dann wird das BSS-Fenster anzeigen, wie viele Neuveröffentlichungen einem Nutzer noch zustehen. Sofern die Beatmap nominiert ist, warnt das Fenster davor, dass die Nominierung durch das Aktualisieren der Beatmap zurückgesetzt wird. Wenn die Beatmap sich auf dem Friedhof befindet, dann weist das Fenster daraufhin, dass die Beatmap in die Kategorie "Ausstehend" verschoben wird.
+
+## Einreichungsoptionen
+
+![](img/bss_submitting-DE.png "Upload-Bildschirm des Beatmap-Einreichungssystems")
+
+Beim Anklicken des Buttons `NEUE Beatmap hochladen!` oder `Beatmap aktualisieren!` hat ein Nutzer die Wahl, ihre Beatmap in die Kategorien `Beatmaps in Arbeit` oder `Vollständige Beatmaps` hochzuladen. WIP-Beatmaps können nicht für das Ranking nominiert werden, während ausstehende Beatmaps es können.
```

Comment:
Just want to confirm if the use of `ihre` here is intentional.

Reply:
No, it's not. Thanks for spotting this.

```suggestion
Beim Anklicken des Buttons `NEUE Beatmap hochladen!` oder `Beatmap aktualisieren!` hat ein Nutzer die Wahl, seine Beatmap in die Kategorien `Beatmaps in Arbeit` oder `Vollständige Beatmaps` hochzuladen. WIP-Beatmaps können nicht für das Ranking nominiert werden, während ausstehende Beatmaps es können.
```